test/integration/utils/PriceMovement.sol

```diff
@@ -11,86 +11,10 @@ library PriceMovementHelper {
     using SafeERC20 for IERC20;
 
     // Price movement configuration
-    uint constant STEPS = 5;
-    uint constant STEP_DELAY = 60 seconds;
-    uint constant MAX_ATTEMPTS = 5;
+    uint constant STEPS = 10;
+    uint constant STEP_DELAY = 300 seconds;
```

Comment:
Setting the step too short meant the TWAP price would be far behind swap price, so you could easily swap out of abailable liquidity, while still below target on TWAP price - which would cause slippage once you tried to actually use the pool later